### PR TITLE
Use namespace variable instead of default namespace value, glusterfs

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>

### CODE REVIEW

File: roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 # oc delete --cascade=false seems broken for DaemonSets.
 # Using curl to talk to the API directly.
 - name: Delete glusterfs daemonset w/o cascade
-  shell: "curl -k -X DELETE https://localhost:8443/apis/extensions/v1beta1/namespaces/glusterfs/daemonsets/glusterfs-storage -d '{\"kind\":\"DeleteOptions\",\"apiVersion\":\"v1\",\"propagationPolicy\":\"Orphan\"}' -H \"Accept: application/json\" -H \"Content-Type: application/json\"  --cert {{ openshift.common.config_base }}/master/admin.crt --key {{ openshift.common.config_base }}//master/admin.key"
+  shell: "curl -k -X DELETE https://localhost:8443/apis/extensions/v1beta1/namespaces/\"{{ glusterfs_namespace }}\"/daemonsets/glusterfs-storage -d '{\"kind\":\"DeleteOptions\",\"apiVersion\":\"v1\",\"propagationPolicy\":\"Orphan\"}' -H \"Accept: application/json\" -H \"Content-Type: application/json\"  --cert {{ openshift.common.config_base }}/master/admin.crt --key {{ openshift.common.config_base }}//master/admin.key"
```
Comment (vrutkovs) [inline roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml]: No need to add an extra quotes here

Comment (SaravanaStorageNetwork) [reply roles/openshift_storage_glusterfs/tasks/glusterfs_upgrade.yml]: > No need to add an extra quotes here

Done

**Comment by jarrpa** [comment]
@vrutkovs As the change you requested was met, I am dismissing your previous review to move the process along.

**Pull request merged**